developing/dev-setup.mdx

```diff
@@ -60,6 +65,14 @@ $ sudo -u postgres psql -c "CREATE USER arroyo WITH PASSWORD 'arroyo' SUPERUSER;
 $ sudo -u postgres createdb arroyo
 ```
 
+On MacOS:
+
+```bash
+$ psql postgres -c "CREATE USER arroyo WITH PASSWORD 'arroyo' SUPERUSER;"
```

Comment:
It indicates which lines are shell commands, otherwise I think it's kind of hard to understand multiple lines all together. It does unfortunately make copying harder. Ideally, the $ would be rendered but would not be selectable (like https://rustup.rs/ for example).